pkg/products/monitoring/prometheusRules.go

```diff
@@ -301,6 +301,33 @@ func (r *Reconciler) newAlertsReconciler(logger l.Logger, installType string) re
 					},
 				},
 			},
+			{
+				AlertName: "test-alerts",
+				Namespace: r.Config.GetOperatorNamespace(),
+				GroupName: "test.rules",
+				Rules: []monitoringv1.Rule{
+					{
+						Alert: "TestFireCriticalAlert",
+						Annotations: map[string]string{
+							"sop_url": "",
+							"message": fmt.Sprintf("This is occasional Test Fire alert from Team SRE"),
+						},
+						Expr:   intstr.FromString("count(kube_secret_info{namespace='redhat-rhoam-operator', secret='cj3cssrec'}) > 0"),
```

Comment:
We really don't want to be hard coding namespaces into any part of the operator. Below is the example of how to get the current installed operators namespace. ie. redhat-rhoam-operator, redhat-rhmi-operator or redhat-rhoami-operator. This would want to be changed here and in the `TestFireWarningAlert` alert
 
```suggestion
						Expr:   intstr.FromString("count(kube_secret_info{namespace=" + r.Config.GetOperatorNamespace() + ", secret='cj3cssrec'}) > 0"),
```

Reply:
Fixed. Thanks